static/src/stylesheets/module/facia/_item.scss

```diff
@@ -311,6 +311,10 @@ $fc-item-gutter: $gs-gutter / 4;
 
 .fc-item__image-container {
     display: none;
+    // Override u-responsive-ratio
+    // Circumvents Chrome bug with positioning, overflow: hidden and columns
+    // https://code.google.com/p/chromium/issues/detail?id=527709
+    overflow: visible;
```

Comment:
I'm not sure why `u-responsive-ratio` sets `overflow: hidden`, but it is used in several places so I don't want to try removing it for the sake of working around this Chrome bug.